model/ftn/ww3_ounf.ftn

```diff
@@ -5104,6 +3596,9 @@
 !/SMC         ENDIF
       ENDIF
 
+      ! ChrisB: Write user global attributes:
+      CALL WRITE_GLOBAL_META(NCID, IRET)
```

Comment:
should you add a CHECK_ERROR call here ?

Reply:
Yup! Done!